scripts/install-dependencies.sh

```diff
@@ -109,19 +109,12 @@ echo "*******************"
 kubectl apply -f "${DEP_DIR}/contour-1.19.1.yaml"
 
 echo "*******************"
-echo "Installing HNC"
+echo "Installing Eirini"
 echo "*******************"
 
-kubectl apply -k ${DEP_DIR}/hnc/cf
-kubectl rollout status deployment/hnc-controller-manager --watch --namespace hnc-system
-while ! kubectl get hncconfigurations.hnc.x-k8s.io config; do
-  sleep 1
-done
-kubectl patch hncconfigurations.hnc.x-k8s.io config --type=merge -p '{"spec":{"resources":[{"mode":"Propagate", "resource": "secrets"}]}}'
+"${SCRIPT_DIR}/generate-eirini-certs-secret.sh" "*.eirini-controller.svc"
```

Comment:
Removing HNC, I ran into `eirini-webhooks-certs` not found error during installation. 
I had to revert the change done as part of PR #1115 to get Eirini to install. 
cc @kieron-dev

Reply:
I don't think reverting those changes is the correct thing to do here. The preferred method for generating webhook certs for eirini-controller-v0.3.0 is using cert-manager, and not by passing `webhooks.ca_bundle`. This is working on the main branch, so what change in this PR is breaking that?

Reply:
I think you may well see something like `MountVolume.SetUp failed for volume "certs" : secret "eirini-webhooks-certs" not found` when starting eirini-controller, but that is just a temporary error if cert-manager is slow to create the certs. Eirini-controller will start up eventually. If that is the problem, you should undo these changes.

Reply:
Thanks @kieron-dev. That was exactly the error i observed on `eirini-controller`. 
The `deploy-on-kind` script kept failing on install of eirini because of it.
cc @matt-royal

Reply:
Thank you for the context, @kieron-dev. We reverted the revert in our newest commit